### PR TITLE
Fix lint scripts to check all targets

### PR DESCRIPTION
**Pull request by jtran**
Prior to this change, our commands didn't actually lint everything.

### CODE REVIEW

**Comment by adamchalmers** [comment]
TBH I don't know what `--all-targets` does, what exactly is a target?

**Comment by jtran** [comment]
From `cargo check --help`:

Screenshot 2024-10-29 at 12 09 06 PM

From https://doc.rust-lang.org/cargo/commands/cargo-check.html:

> `--all-targets`
>    Check all targets. This is equivalent to specifying `--lib --bins --tests --benches --examples`.

File: src/wasm-lib/justfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ new-test name:
     TWENTY_TWENTY=overwrite cargo nextest run --test executor -E 'test(=visuals::{{name}})'
 
 lint:
-    cargo clippy --all --tests --benches -- -D warnings
+    cargo clippy --all --all-targets --tests --benches -- -D warnings
```
Comment (adamchalmers) [inline src/wasm-lib/justfile:7]: If --all-targets includes tests and benches, then we should remove --tests and --benches right? They're unnecessary and implied by --all-targets?

Comment (jtran) [reply src/wasm-lib/justfile]: Done.

**Review by adamchalmers — APPROVED** [review]

**Pull request merged**